{"uid":"1bb17b679e4ff2f1","name":"Sign in with valid and invalid data","fullName":"vizsgaremek.automated_tests.test_conduit.TestSignInAndSignOut#test_sign_in_with_csv","historyId":"152f52f10c5dbdcf7183d56cb503c64b","time":{"start":1682252865426,"stop":1682252865426,"duration":0},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'vizsgaremek/test/datasource_users.csv'","statusTrace":"self = <test_conduit.TestSignInAndSignOut object at 0x7f4291e8c340>\n\n    @allure.id(\"TC-03\")\n    @allure.title(\"Sign in with valid and invalid data\")\n    def test_sign_in_with_csv(self):\n        wait = WebDriverWait(self.browser, 10)\n>       for user in get_users_from_file():\n\nvizsgaremek/automated_tests/test_conduit.py:73: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    def get_users_from_file():\n>       with open('vizsgaremek/test/datasource_users.csv', 'r', encoding='UTF-8') as datafile:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'vizsgaremek/test/datasource_users.csv'\n\nvizsgaremek/automated_tests/main_functions.py:34: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_xunit_setup_method_fixture_TestSignInAndSignOut","time":{"start":1682252864531,"stop":1682252865425,"duration":894},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"afterStages":[{"name":"_xunit_setup_method_fixture_TestSignInAndSignOut::0","time":{"start":1682252865435,"stop":1682252865491,"duration":56},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"as_id","value":"TC-03"},{"name":"parentSuite","value":"vizsgaremek.automated_tests"},{"name":"suite","value":"test_conduit"},{"name":"subSuite","value":"TestSignInAndSignOut"},{"name":"host","value":"fv-az349-463"},{"name":"thread","value":"3150-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"vizsgaremek.automated_tests.test_conduit"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"1bb17b679e4ff2f1.json","parameterValues":[]}